tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -452,9 +453,9 @@ private String constructExampleHtml(String xpath, String value) {
             result = handleLabelPattern(parts, value);
         }
         if (result != null) {
-            if (!typeIsEnglish) {
-                result = addTransliteration(result, value);
-            }
+//            if (!typeIsEnglish) {
+//                result = addTransliteration(result, value);
+//            }
```

Comment:
this could use clarification -- if not temporary, best to remove commented-out code